app/packages/looker/src/processOverlays.ts

```diff
@@ -30,6 +32,25 @@ const processOverlays = <State extends BaseState>(
 
     if (!(overlay.field && overlay.field in bins)) continue;
 
+    // todo: find a better approach / place for this.
+    // for instance, this won't work in detection overlay, where
+    // we might want the bounding boxes but masks might not have been loaded
```

Comment:
I think these instances can be filtered out during overlay creation. But this is a good place for now 👍